ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/Desktop.css

```diff
@@ -183,9 +213,10 @@
 .userContextMenu {
     background-color: white;
     outline: none;
-    border: solid #0098AA;
+    border: solid #0971AB;
     padding: 1px;
     margin-top: 5px;
+    font-family: TextaBold; /* TextaRegular a little hard to read on the menu items*/
```

Comment:
This may not be a thing to do right now, but this should have a fallback font in these sorts of cases. The `html, body` declaration below has a bunch of fallbacks but these specific overrides don't.